opentelemetry-api/src/opentelemetry/metrics/__init__.py

```diff
@@ -26,15 +26,15 @@
 
 
 """
+import enum
+
 from abc import ABC, abstractmethod
-from typing import Dict, List, Tuple, Type, Union
+from typing import Callable, Generic, Optional, Tuple, Type, TypeVar, Union
+
+from opentelemetry.util import loader
 
-from opentelemetry.metrics.handle import (
-    CounterHandle,
-    GaugeHandle,
-    MeasureHandle,
-)
-from opentelemetry.trace import SpanContext
+MetricKind = Union[Type["Counter"], Type["Gauge"], Type["Measure"]]
+ValueType = TypeVar('ValueType', int, float)
```

Comment:
is int 64 bits? Use the 64 bits version for integer and float :) - My python skills ~ 0 :)

Reply:
Integers have unlimited precision in Python 3. As for float, it is usually implemented with `double` in C.

Source: https://docs.python.org/3/library/stdtypes.html#typesnumeric